src/hal/getpinmap_disco_l072cs_lrwan1.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+
+Module:  getconfig_catena4610.cpp
```

Comment:
comment needs to be updated for the board.

Reply:
Thanks for catching these @terrillmoore, I've updated them!